cli/support_test.go

```diff
@@ -141,22 +155,44 @@ func assertBundleContents(t *testing.T, path string) {
 		case "network/tailnet_debug.html":
 			bs := readBytesFromZip(t, f)
 			require.NotEmpty(t, bs, "tailnet debug should not be empty")
-		case "network/netcheck_local.json", "network/netcheck_remote.json":
-			// TODO: setup fake agent?
-			bs := readBytesFromZip(t, f)
-			require.NotEmpty(t, bs, "netcheck should not be empty")
+		case "network/netcheck.json":
+			var v codersdk.WorkspaceAgentConnectionInfo
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "connection info should not be empty")
 		case "workspace/workspace.json":
 			var v codersdk.Workspace
 			decodeJSONFromZip(t, f, &v)
 			require.NotEmpty(t, v, "workspace should not be empty")
 		case "workspace/build_logs.txt":
 			bs := readBytesFromZip(t, f)
 			require.Contains(t, string(bs), "provision done")
-		case "workspace/agent.json":
+		case "agent/agent.json":
 			var v codersdk.WorkspaceAgent
 			decodeJSONFromZip(t, f, &v)
 			require.NotEmpty(t, v, "agent should not be empty")
-		case "workspace/agent_startup_logs.txt":
+		case "agent/listening_ports.json":
+			var v codersdk.WorkspaceAgentListeningPortsResponse
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "agent listening ports should not be empty")
+		case "agent/logs.txt":
+			bs := readBytesFromZip(t, f)
+			require.NotEmpty(t, bs, "logs should not be empty")
+		case "agent/magicsock.html":
+			bs := readBytesFromZip(t, f)
+			require.NotEmpty(t, bs, "agent magicsock should not be empty")
+		case "agent/manifest.json":
+			var v agentsdk.Manifest
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "agent manifest should not be empty")
+		case "agent/peer_diagnostics.json":
+			var v *tailnet.PeerDiagnostics
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "peer diagnostics should not be empty")
```

Comment:
How well does NotEmpty work with pointers? Should it be dereferenced perhaps?

Reply:
Added a test helper to assert not nil and not empty.